polaris-react/src/components/FullscreenBar/FullscreenBar.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../styles/common';
+
+.FullscreenBar {
+  position: relative;
+  display: flex;
+  height: top-bar-height();
+  box-shadow: var(--p-shadow-top-bar);
+  background-color: var(--p-surface);
+}
+
+.BackAction {
+  display: flex;
+  align-items: center;
+  color: var(--p-text);
+  fill: var(--p-icon);
+  padding-left: var(--p-space-2);
+  padding-right: var(--p-space-2);
+  border-radius: 0;
+  border-right: var(--p-border-divider);
+  border-bottom: var(--p-border-transparent);
+  border-left: var(--p-border-transparent);
+  border-top: var(--p-border-transparent);
```

Comment:
Can we just specify border right?

Reply:
Unfortunately. it has a black border by default that I need to override.  But I can do it with less lines.